mpifileutils.rpmlintrc

```diff
@@ -4,3 +4,5 @@ from Config import *
 # https://github.com/rpm-software-management/rpmlint/issues/73
 addFilter(r'mpifileutils-.+-devel\..+: W: only-non-binary-in-usr-lib')
 
+# https://github.com/hpc/mpifileutils/issues/567
+addFilter(r'mpifileutils-.+\.x86_64: E: missing-call-to-setgroups-before-setuid \/usr\/lib64\/.+/bin/dcp')
```

Comment:
It's interesting that this is actually working with the missing RE escapes.  But we should also prevent it from matching, say, `/usr/lib64/openmpi/bin/dcpfoo':

```suggestion
addFilter(r'^mpifileutils-.+\.x86_64: E: missing-call-to-setgroups-before-setuid \/usr\/lib64\/.+\/bin\/dcp$')
```